### ISSUE TITLE
irl crouching makes the player constantly twitching up and down

### ISSUE THREAD

**Issue by Hoshi82** [DSprtn/GTFO_VR_Plugin#10]
As topic says. Also when crouch in irl and get back up I am still in crouching position and have to use the crouch function on the grip button. I checked the cfg and calibrated my height but that didn't fix the issue.

**Comment by DSprtn** [comment]
I can reproduce the twitch,  I'll have a look at it. Crouching up and down IRL seems to be working correctly though. I'm guessing it might have to do with having crouching set as a toggle in SteamVR input settings. I'll look into that soon.

**Comment by Hoshi82** [comment]
Thanks :)
Yes the second time I played irl crouching worked fine (except the twitching :) ).

**Comment by Hoshi82** [comment]
Before starting a new topic. It seems with the new update the terminals won't let me type anything. I can hit the "done" key on the virtual keyboard and that works but trying to enter letters ect does not.

**Comment by DSprtn** [comment]
It's better to make a new topic for every other new issue ^^

Having said that, I cannot reproduce the terminal not working. I do know that SteamVR itself sometimes bugs out so try restarting it and trying it again?

**Comment by DSprtn** [comment]
Crouching will be fixed in the next update, terminal has been fixed in v0.502!
